src/WaterPokemon.java

```diff
@@ -0,0 +1,34 @@
+package src;
+
+public class WaterPokemon extends Pokemon {
+
+    String attack;
+    public WaterPokemon(String name, int weight, int height, int hp, String sound, String food, String attack) {
+        super(name, weight, height, hp, sound, food);
+        this.attack = attack;
+    }
+
+    public void speaks(String sound) {
+        System.out.println(sound);
+
+    }
+
+    @Override
+    public void eats(String food) {
+        System.out.println(food);
+
+    }
+
+    public void hydroPump(String attack) {
+        System.out.println(attack);
```

Comment:
This method doesn't need a parameter. You can either print the String "hydropump" or "this.name + "uses hydropump"" OR you can use the attack variable you defined on line 5.